File: Moxion/Common/Enumerations/MotionPhase.cs
namespace Moxion.Common.Enumerations;

public enum MotionPhase
{
  None,
  AccelerationWithPositiveJerk,
  ConstantAcceleration,
  AccelerationWithNegativeJerk,
  ConstantVelocity,
  DecelerationWithNegativeJerk,
  ConstantDeceleration,
  DecelerationWithPositiveJerk,
}

File: Moxion.Presentation.Web/ClientApp/src/models/enumerations/MotionPhase.ts
type MotionPhase =
    'None'
    | 'AccelerationWithPositiveJerk'
    | 'ConstantAcceleration'
    | 'AccelerationWithNegativeJerk'
    | 'ConstantVelocity'
    | 'DecelerationWithNegativeJerk'
    | 'ConstantDeceleration'
    | 'DecelerationWithPositiveJerk'

export default MotionPhase